streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableAggregateTest.java

```diff
@@ -275,53 +226,17 @@ public void testCountWithInternalStore() {
         final StreamsBuilder builder = new StreamsBuilder();
         final String input = "count-test-input";
 
+        final MockProcessorSupplier<String, Object> supplier = new MockProcessorSupplier<>();
```

Comment:
Why we cannot continue to reuse a single `MockProcessorSupplier`?

Reply:
Ah, I misread it. I thought they were all reusing the processor, somehow, which I thought was risky. Since it's just the supplier, I'll move it back.